CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [0.4.9] - SNAPSHOT
 ### Changed
-- Updated swagger example values (#139)
+- Updated swagger example values
```

Comment:
No. Please keep the (#139) on the previous line (otherwise you are undoing the merge)